spec/rubocop/cop/lint/useless_else_without_rescue_spec.rb

```diff
@@ -1,19 +1,6 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::Lint::UselessElseWithoutRescue, :config do
-  context 'with `else` without `rescue`', :ruby25 do
-    it 'registers an offense' do
-      expect_offense(<<~RUBY)
-        begin
-          do_something
-        else
-        ^^^^ `else` without `rescue` is useless.
-          handle_unknown_errors
-        end
-      RUBY
-    end
-  end
```

Comment:
This fails on Ruby 2.6+, and we don't run specs on CI for them.
Is this spec still useful? Is it still possible to run RuboCop specs locally on Ruby 2.5?